SP23.P02.Web/Controllers/StationsController.cs

```diff
@@ -74,8 +80,14 @@ public ActionResult<TrainStationDto> UpdateStation(int id, TrainStationDto dto)
             return NotFound();
         }
 
+        if (!User.IsInRole(Role.Admin))
+        {
+            return Forbid();
+        }
+
         station.Name = dto.Name;
         station.Address = dto.Address;
+        station.ManagerId = dto.ManagerId;
```

Comment:
Only admins should be able to change the manager id of the station, so you could probably just put this in an if statement with `User.IsInRole(Role.Admin)`.

I don't believe this actually affects any tests, but it is technically part of the guidelines for this phase.